user-management.yaml

```diff
@@ -459,6 +459,157 @@ paths:
+        "422":
+          description: |
+            Unprocessable Entity   
+            types: LoginTokenSignatureInvalid
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GenericError'
+    put:
+      tags:
+      - "User Management"
+      summary: "Set a new profile picture"
+      description: |
+        Updates a user with the provided profile picture
+      operationId: "putImage"
+      security:
+        - uc4_token_login: [] 
+        - uc4_token_login_header: []
+      parameters:
+      - name: "username"
+        in: "path"
+        required: true
+        schema:
+          type: "string"
+      requestBody:
+        description: "Byte string of the new profile picture"
+        required: true
+        content:
+          image/*:
+            schema:
+              type: "string"
+              format: binary
+            
+      responses:
+        "200":
+          description: "OK"
+        "400":
+          description: | 
+            Bad Request  
+            types: MalformedLoginToken, MultipleAuthorizationError, Deserialization, MissingHeader
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - $ref: '#/components/schemas/GenericError'
+                  - $ref: '#/components/schemas/DetailedError'
+        "401":
+          description: |
+            Unauthorized  
+            types: JwtAuthorization, LoginTokenExpired
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GenericError'
+        "403":
+          description: |
+            Forbidden  
+            types: OwnerMismatch
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GenericError'
+        "404":
+          description: |
+            Not Found  
+            types: KeyNotFound
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GenericError'
+        "413":
+          description: |
+            Payload Too Large
+            types: EntityTooLarge
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GenericError'
+        "415":
+          description: |
+            Unsupported Media Type 
+            types: UnsupportedMediaType
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GenericError'
+        "422":
+          description: |
+            Unprocessable Entity  
+            types: LoginTokenSignatureInvalid
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - $ref: '#/components/schemas/GenericError'
+                  - $ref: '#/components/schemas/DetailedError'
```

Comment:
```suggestion
            Unprocessable Entity  
            types: LoginTokenSignatureInvalid
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/GenericError'
```

Reply:
We may add this to the API later. At the moment we cannot implement this feature, because users cannot be added to courses. Therefore, we should not add this to the API just now.